content/javascript/concepts/arrays/terms/copyWithin/copyWithin.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: '.copyWithin()'
+Description: 'Copies part of an array to another location in the same array and returns it without modifying its length.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.copyWithin()` copies part of an array to another location in the same array and returns it without modifying its length.
+
+## Syntax
+
+The `.copyWithin()` method is a mutating method. It does not alter the length of the array, but it will change the content of the array and create new properties or delete existing properties, if necessary.
+
+```js
```

Comment:
We generally have syntax examples in a ```pseudo block.

Reply:
Ok, sure, I will change that. However, files like foreach and filter have the syntax written in js bock. In the future, do you think I should change them to a pseudo block in another branch?

Reply:
@FireBird1998, it's a recent style decision. It's not urgent, but you could go ahead and do so.